.github/workflows/salt-sls-files-checks.yml

```diff
@@ -0,0 +1,21 @@
+name: Check the content of Salt SLS files
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize, reopened]
```

Comment:
You probably don't want this workflow to run on `edited`, as it would be triggered whenever the PR **description** is changed! The remaining event types should be just fine and this is actually the default for `pull_request`, i.e. you can completely omit the `types`. Instead you probably want to add a `paths` filter, see my updated version of [the rubocop workflow](https://github.com/uyuni-project/uyuni/blob/master/.github/workflows/rubocop.yml).